src/internal.c

```diff
@@ -13431,10 +13431,12 @@ static int DoDtlsHandShakeMsg(WOLFSSL* ssl, byte* input, word32* inOutIdx,
             }
             *inOutIdx += ssl->keys.padSz;
             ret = 0;
+            #ifndef WOLFSSL_DTLS_RESEND_ONLY_TIMEOUT
```

Comment:
This is a naming issue more than anything, and a little bit of a nitpick. Could this flag be renamed `WOLFSSL_DTLS_NO_OOO_RESEND`? "OOO" is for out-of-order.

Reply:
Yeah no problem.
> Richard, if we are going to change the name, we need to do it now. Names stick. My name's only Jared because Gavin called me that on my first day. My real name is Donald.

Reply:
It was determined that the current name is better suited.